README.md

```diff
@@ -84,8 +84,8 @@ If you'd like to chat, we have a [community slack](http://amserializers.herokuap
 Thanks!
 
 ## Documentation
-- [0.10 (master) Documentation](https://github.com/rails-api/active_model_serializers/tree/master)
-  - [![API Docs](http://img.shields.io/badge/yard-docs-blue.svg)](http://www.rubydoc.info/gems/active_model_serializers/0.10.2)
+- [0.10 (master) Documentation](https://github.com/rails-api/active_model_serializers/commit/cd09e890064d2611f2977975163224e12e119568)
```

Comment:
should be `https://github.com/rails-api/active_model_serializers/tree/v0.10.3`  but really we need a note like:

```
**If you're viewing this at https://github.com/rails-api/active_model_serializers,
 you're reading the documentation for the master branch.
 View documentation for the latest release.... etc
```